microsoft/testsuites/vm_extensions/linux_patch_extension.py

```diff
@@ -54,6 +54,21 @@ def _verify_vm_agent_running(node: Node, log: Logger) -> None:
     ).is_true()
 
 
+def _assert_status_file_result(status_file: Any) -> None:
+    if status_file["error"]["code"] == "PACKAGE_LIST_TRUNCATED":
```

Comment:
can we check the status code in a loop instead of waiting the operations till time out? since I found when it is in CompletedWithWarnings situation, we have to wait till time out, it costs 1.5 hours.

https://github.com/microsoft/lisa/blob/b17820284813f1d656a4b6814cffca004ffd160f/microsoft/testsuites/vm_extensions/linux_patch_extension.py#L144

Reply:
I ran bvt on image marketplace_image:redhat rhel 8-lvm 8.5.2022031405, it didn't take up to 1.5hours, however if we do encounter a failure, then it will take up to 1.5hour.